hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -490,9 +489,9 @@ protected void validateAdapterSchema(
             case SUCCESS:
                 return Response.ok().build();
             case ADAPTER_NOT_FOUND:
-                return Response.status(403).entity("").build();
+                return ApiErrorUtils.badRequest("Adapter not found");
             case INTERNAL_ERROR:
-                return Response.serverError().build();
+                return Response.serverError().entity("").build();
```

Comment:
thats not a legal json, guess you want to use the `INTERNAL_ERROR_PLEASE_CHECK_LOGS`?

Reply:
Yep!